Localization.ruRU.lua

```diff
@@ -570,7 +570,7 @@ PawnLocal.TooltipParsing = {
 	["Haste2"] = "^%+?# к показателю скорости$",
 	["HasteRating"] = "^Если на персонаже: Повышает рейтинг скорости боя на #%.$",
 	["HasteRating2"] = "^UNUSED$",
-	["HasteRatingShort"] = "^%+?# к рейтингу? скорости$",
+	["HasteRatingShort"] = "^%+?# рейтингу? скорости$",
```

Comment:
Can you check to see if this works as well? That could help avoid having to make another change again when Mists Classic or the next round of Burning Crusade Classic comes around...

```suggestion
	["HasteRatingShort"] = "^%+?# к? ?рейтингу? скорости$",
```

Reply:
Nope, "^%+?# к? ?рейтингу? скорости$" doesn't work.